content/en/database_monitoring/setup_sql_server/selfhosted.md

```diff
@@ -0,0 +1,195 @@
+
+## Grant the Agent access
+
+The Datadog Agent requires read-only access to the database server in order to collect statistics and queries.
+
+Create a read-only login to connect to your server:
+
+```SQL
+CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
+CREATE USER datadog FOR LOGIN datadog;
+GRANT SELECT on sys.dm_os_performance_counters to datadog;
+GRANT VIEW SERVER STATE to datadog;
+```
+
+To collect file size metrics per database, ensure the user you created (`datadog`) has [connect permission][5] access to your databases by running:
+
+```SQL
+GRANT CONNECT ANY DATABASE to datadog;
+```
+
+Additionally to collect AlwaysOn and `sys.master_files` metrics, grant the following additional permission:
+
+```SQL
+GRANT VIEW ANY DEFINITION to datadog;
+```
+
+## Install the Agent
+
+If you haven't already installed the Agent for your SQL Server database host, see the [Agent installation instructions][6].
+
+1. Edit the Agent's `conf.d/sqlserver.d/conf.yaml` file to point to your `host` / `port` and set the hosts to monitor. See the [sample sqlserver.d/conf.yaml][7] for all available configuration options.
+
+{{< tabs >}}
+{{% tab "Windows" %}}
+
+    ```yaml
+    init_config:
+    instances:
+      - dbm: true
+        host: '<SQL_HOST>,<SQL_PORT>'
+        username: datadog
+        password: '<PASSWORD>'
+        connector: odbc  # Alternative is 'adodbapi'
+        driver: SQL Server
+        tags:  # optional
+          - 'service:<CUSTOM_SERVICE>'
+          - 'env:<CUSTOM_ENV>'
+    ```
+
+{{% /tab %}}
+{{% tab "Linux" %}}
+
+    For SQL Server installations running on Linux hosts, use the `odbc` connector and specify the proper driver as indicated in the `odbcinst.ini` file.
+
+    ```yaml
+    init_config:
+    instances:
+      - dbm: true
+        host: '<SQL_HOST>,<SQL_PORT>'
+        username: datadog
+        password: '<PASSWORD>'
+        connector: odbc
+        driver: '<Driver from the `odbcinst.ini` file>'
+        tags:  # optional
+          - 'service:<CUSTOM_SERVICE>'
+          - 'env:<CUSTOM_ENV>'
+    ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+  The `service` and `env` tags allow you to link your database telemetry to other telemetry through a common tagging scheme. To see how these tags are used throughout Datadog, see the documentation on [unified service tagging][8].
+
+  See the [example check configuration][7] for a comprehensive description of all options, including how to use custom SQL queries to create your own metrics.
+
+  **Note**: The (default) provider `SQLOLEDB` is being deprecated. To use the newer `MSOLEDBSQL` provider, set the `adoprovider` variable to `MSOLEDBSQL` in your `sqlserver.d/conf.yaml` file after having downloaded the new provider from [Microsoft][9]. It is also possible to use the Windows Authentication and not specify the username/password with:
+
+  ```yaml
+  connection_string: "Trusted_Connection=yes"
+  ```
```

Comment:
Just trying to make sure everything up to this point is indented/included in step one with three spaces.
~~~suggestion
   ```yaml
   connection_string: "Trusted_Connection=yes"
   ```
~~~